.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,145 @@
+          - `expo start -c`
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Required information
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please provide a clear and concise description of what the bug is. Include screenshots or a video if needed. Tell us what were you expecting to happen instead of what is happening now.
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Provide a detailed list of steps that reproduce the issue.
+      placeholder: |
+        1.
+        2.
+        3.
+    validations:
+      required: true
+
+  - type: input
+    id: repro
+    attributes:
+      label: Snack or a link to a repository
+      description: |
+        Please provide a Snack (https://snack.expo.io/) or a link to a repository on GitHub under your username that reproduces the issue.
+        Here are some tips for providing a minimal example: https://stackoverflow.com/help/mcve.
+        Issues without a reproduction are likely to stale.
+      placeholder: Link to a Snack or a GitHub repository
+    validations:
+      required: true
+
+  - type: input
+    id: reanimated-version
+    attributes:
+      label: Reanimated version
+      description: What version of react-native-reanimated are you using?
+      placeholder: 2.9.0
+    validations:
+      required: true
+
+  - type: input
+    id: react-native-version
+    attributes:
+      label: React Native version
+      description: What version of react-native are you using?
+      placeholder: 0.69.0
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: Platforms
+      description: On what platform your application is running on?
+      multiple: true
+      options:
+        - Android
+        - iOS
+        - Web
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Additonal information
+
+        Providing as much information as possible greatly helps us with reproducting the issues.
+
+  - type: dropdown
+    id: runtime
+    attributes:
+      label: JavaScript runtime
+      description: What runtime is your application using?
+      options:
+        - JSC
+        - Hermes
+        - V8
+
+  - type: dropdown
+    id: workflow
+    attributes:
+      label: Workflow
+      description: How your application is managed? Not sure? Read [this part of Expo documentation](https://docs.expo.dev/introduction/managed-vs-bare/)
+      options:
+        - React Native (without Expo)
+        - Expo bare workflow
+        - Expo managed workflow
+
+  - type: dropdown
+    id: architecture
+    attributes:
+      label: Architecture
+      description: What React Native architecture your application is running on?
+      options:
+        - Paper (current)
+        - Fabric (new)
```

```suggestion
        - Paper (Old Architecture)
        - Fabric (New Architecture)
```

> The goal of the New Architecture is to solve some of the issues that afflicted the Old Architecture in terms of performance and flexibility.

https://reactnative.dev/docs/next/the-new-architecture/why